solidity/test/random_beacon_operator/TestSlashing.js

```diff
@@ -129,7 +129,7 @@ contract('KeepRandomBeaconOperator/Slashing', function(accounts) {
   it("should seize no more than available operator's amount", async () => {
     let tattletaleBalanceBeforeSeizing = await token.balanceOf(tattletale)
     
-    let amountToSeize = largeStake.add(web3.utils.toBN(100)); // 400000000000000000000100
+    let amountToSeize = largeStake.add(web3.utils.toBN(100));
```

Comment:
Can we add `// 200000000000000000000100` comment back here? It helps to understand where does the equation for expected tattletale reward below.